docs/modules/ROOT/pages/tutorials/demo-app/appcat-nextcloud.adoc

```diff
@@ -0,0 +1,128 @@
+
+== Requirements
+
+This tutorial requires an {product}, a https://github.com[GitHub] account.
+
+== Add the https://products.vshn.ch/appcat/index.html[VSHN Application Catalog] claim
+
+To provision Nextcloud, we need a https://products.vshn.ch/appcat/nextcloud.html[VSHNNextcloud] claim. The documentation for VSHNNextcloud can be found here: https://docs.appcat.ch/vshn-managed/nextcloud/index.html[Nextcloud by VSHN].
+We start with a very simple setup:
+
+.VSHNNextcloud
+[source,yaml]
+----
+apiVersion: vshn.appcat.vshn.io/v1
+kind: VSHNNextcloud
+metadata:
+  name: appcat-demo-nextcloud-appuio-simple # <1>
+  namespace: vshn2-demo-nextcloud-appuio-simple-prod # <2>
+spec:
+  parameters:
+    service:
+      fqdn:
+      - appuio-nextcloud-demo.apps.cloudscale-lpg-2.appuio.cloud # <3>
+      version: "32" # <4>
+    size: # <5>
+      plan: standard-2
+  writeConnectionSecretToRef:
+    name: nextcloud-creds # <6>
+----
+<1> Instance name
+<2> The namespace where the object will be created
+<3> Your fully qualified domain name. You should be able to change the DNS record for this domain.
+<4> Nextcloud version
+<5> Size of the Nextcloud instance. See https://docs.appcat.ch/vshn-managed/nextcloud/plans.html[Plans and Sizing] for more information.
+<6> Secret where the connection details are provisioned. No secret with this name should exist in this namespace before creation.
+
+== Put the claim under version control
+
+To keep track of what's deployed, we put this claim into our Git repository.
+For that, you need to create a https://github.com/new[new Git repository] on GitHub.
+Now, we store the claim from before under `helm/templates/nextcloud.yaml`.
+You can see the result in the https://github.com/vshn/appcat-demo-nextcloud-appuio-simple/tree/main/helm/templates[example repository].
+We also need some boilerplate for Helm; you can also find this in the https://github.com/vshn/appcat-demo-nextcloud-appuio-simple/tree/main/helm[example repository].
+
+== Deploy the claim
+
+If you are a VSHN customer, you can generate the pipeline to deploy Nextcloud in
+xref:#appflow-helm[AppFlow].
+
+Otherwise, you can copy the `.github/workflows` directory from the https://github.com/vshn/appcat-demo-nextcloud-appuio-simple/tree/main/.github[example repository] and
+adapt it where needed.
+For the pipeline to work, you need to add a kubeconfig to your pipeline.
+First, you need to make sure you have the `oc` utility. You can download it from
+the https://console.cloudscale-lpg-2.appuio.cloud/command-line-tools[OpenShift console command line tools].
+Then you can show the command to create your kubeconfig in the https://oauth-openshift.apps.cloudscale-lpg-2.appuio.cloud/oauth/token/display?code=sha256~xhC3o8qafyrrISMXZhKRt9uhIf-687n7LCHL6jpYl5k&state=[OpenShift console].
+
+.Generate kubeconfig for pipeline
+[source,shell]
+----
+KUBECONFIG=./config-for-pipeline
+oc login ...
```

Comment:
why not just `oc login --web --server https://api.<zone>.appuio.cloud:6443`? That should pop up a login screen or immediately fetch a token via local browser integration.